projects/packages/my-jetpack/_inc/data/products/use-install-standalone-plugin.ts

```diff
@@ -0,0 +1,34 @@
+import { __, sprintf } from '@wordpress/i18n';
+import { REST_API_SITE_PRODUCTS_ENDPOINT } from '../constants';
+import useSimpleMutation from '../use-simple-mutation';
+import useProduct from './use-product';
+
+const useInstallStandalonePlugin = ( productId: string ) => {
+	const { detail, refetch } = useProduct( productId );
+
+	const { mutate: install, isPending } = useSimpleMutation(
+		'installPlugin',
+		{
+			path: `${ REST_API_SITE_PRODUCTS_ENDPOINT }/${ productId }/install-standalone`,
+			method: 'POST',
+		},
+		{
+			onSuccess: async () => {
+				await refetch();
+			},
+		},
+		null,
```

Comment:
As we grow number of params we might consider moving them to the "options" object instead, as part of the cleanup.

Reply:
Yeah we'll see how it looks after we are done migrating, I could see that being better